src/main/java/cz/cvut/kbss/termit/model/Vocabulary.java

```diff
@@ -236,13 +237,23 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "Vocabulary{" +
-                getLabel() +
-                " " + Utils.uriToString(getUri()) +
-                ", glossary=" + glossary +
-                (importedVocabularies != null ?
-                 ", importedVocabularies = [" + importedVocabularies.stream().map(Utils::uriToString).collect(
-                         Collectors.joining(", ")) + "]" : "") +
-                '}';
+        final StringBuilder builder = new StringBuilder();
```

Comment:
Don't use `StringBuilder` unless constructing some complex string (or in a loop), the JVM is able to optimize string concatenation as necessary.